### ISSUE TITLE
Add an option to be able to turn avatars off

### ISSUE THREAD

**Issue by Krzypis** [vector-im/riot-web#4403]
This is rather annoying for people who prefer compact view. We want to determine the sender by their nickname, not fancy avatars.

**Comment by grahamperrin** [comment]
> … sender … 

Also I'm happy with the _room_ avatars removed from the sidebar: 

![2017-07-30 10 50 27 overview](https://user-images.githubusercontent.com/192271/28752467-2275171c-7518-11e7-8abc-89ef8411a5e2.png)

> … compact … 

Not only for compactness. Without the distractions of apparently random coloured circles, it's easier to focus on content – on messages: 

![2017-07-30 10 54 15 full screen](https://user-images.githubusercontent.com/192271/28752469-3a6bf53e-7518-11e7-91f9-031f6c249a35.png)

A step further, without the room avatar at the head of the message column: 

![2017-07-30 11 03 35 without mx_roomheader_avatar](https://user-images.githubusercontent.com/192271/28752475-72e51378-7518-11e7-86c6-a3a0abb95b65.png)

## With [Stylish](https://addons.mozilla.org/firefox/addon/stylish/)

```text
@-moz-document domain("riot.im") {
  .mx_RoomTile_avatar {
    display: none !important;
  }
  .mx_RoomHeader_avatar {
    display: none !important;
  }
  .mx_EventTile_avatar {
    display: none !important;
  }
}
```

## Related

- [Adapt better to narrow screens · Issue #1633 · vector-im/riot-web](https://github.com/vector-im/riot-web/issues/1633)
- [Minimize UI noise in message feed · Issue #4317 · vector-im/riot-web](https://github.com/vector-im/riot-web/issues/4317)

**Comment by grahamperrin** [comment]
Plus, removal of avatars from pills (and with the dark theme, I prefer something other than green for _linkified_): 

```text
@-moz-document domain("riot.im") {
  .mx_RoomTile_avatar {
    display: none !important;
  }
  .mx_RoomHeader_avatar {
    display: none !important;
  }
  .mx_EventTile_avatar {
    display: none !important;
  }
  .mx_UserPill .mx_BaseAvatar {
    display: none !important;
  }
  .linkified {
    color: rgb(0, 255, 255) !important;
  }
}
```

![2017-07-30 15 14 13](https://user-images.githubusercontent.com/192271/28754281-59d627ce-753a-11e7-82bb-b3a05d8a1177.png)

![2017-07-30 15 14 47](https://user-images.githubusercontent.com/192271/28754366-8dbba89c-753b-11e7-8877-9aa4b8a34bde.png)

### Related

- [Support inlined mentions properly · Issue #2980 · vector-im/riot-web](https://github.com/vector-im/riot-web/issues/2980)
- [@Mentions TODO · Issue #4640 · vector-im/riot-web](https://github.com/vector-im/riot-web/issues/4640)

**Comment by dralley** [comment]
Yes, please.  Any information or options that you can get to by clicking on an avatar is available with little effort from the "room members" panel, so for me personally the benefit of the avatar is significantly outweighed by the added distraction 99% of the time.

Especially in the "compact IRC-style UI layout" option where it's extra-useless.  

![Screenshot from 2020-10-14 12-19-02](https://user-images.githubusercontent.com/2751996/96017409-fb6a0c80-0e17-11eb-8de8-32997631393b.png)

It'd definitely be more IRC-like if it was hidden :)
